### PR TITLE
[READY] Fixes floor cluwne locker check and some other exploits

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
fix: Floor cluwne now properly checks the locker you're in in it's Life proc and attack stage.
fix: Floor cluwne event now works properly.
fix: Floor cluwnes can no longer be seen on HUDs.
tweak: Floor cluwnes can be seen by ghosts for easy observing.
tweak: You no longer scream all the time when being dragged by a floor cluwne.
tweak: You will now have an instinctive chance of calling for help when a floor cluwne pulls you under...
/:cl:

## About The Pull Request
Basically a maintainer forgot to review properly and merged so I had to make this.  I also felt like making your character scream "help me!!"  for a dramatic effect. 

## Why It's Good For The Game
Because bug fixes, less annoying screaming and spooky addition.

### CODE REVIEW

File: hippiestation/code/modules/mob/living/simple_animal/floor_cluwne.dm [hunk 1]
```diff
@@ -143,6 +139,14 @@ GLOBAL_VAR_INIT(floor_cluwnes, 0)
 /mob/living/simple_animal/hostile/floor_cluwne/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, safety = 0, tesla_shock = 0, illusion = 0, stun = TRUE)//prevents runtimes with machine fuckery
 	return FALSE
 
+/mob/living/simple_animal/hostile/floor_cluwne/proc/Found_You()
+	for(var/obj/structure/closet/hiding_spot in orange(7,src))
+		if(current_victim.loc == hiding_spot)
+			hiding_spot.bust_open()
+			current_victim.Paralyze(40)
+			to_chat(current_victim, "<span class='warning'>You can't hide...</span>")
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/simple_animal/floor_cluwne.dm]: Most floor cluwne lines are written backwards, might wanna reverse this text.

File: hippiestation/code/modules/mob/living/simple_animal/floor_cluwne.dm [hunk 2]
```diff
@@ -344,8 +349,10 @@ GLOBAL_VAR_INIT(floor_cluwnes, 0)
 		if(do_after(src, 5, target = H))
 			step_towards(H, src)
 			playsound(H, pick('hippiestation/sound/effects/bodyscrape-01.ogg', 'hippiestation/sound/effects/bodyscrape-02.ogg'), 20, 1, -4)
-			H.emote("scream")
+			if(prob(50))
+				H.emote("scream")
 			if(prob(25))
+				H.forcesay("HELP ME!!")
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/mob/living/simple_animal/floor_cluwne.dm]: Suggestion: have a list of possible panic lines, for example "OH GOD!!" "ITS GOT ME!!" "OH SHIT NO!!"

Comment (yoyobatty) [reply hippiestation/code/modules/mob/living/simple_animal/floor_cluwne.dm]: Interesting...

**Comment by yoyobatty** [comment]
Waiting on 1 more commit, don't merge just yet.

**Review by GuyonBroadway — APPROVED** [review]
Dis loogs ogey

**Comment by Jujumatic** [comment]
Why didn't gayon merge reee

**Pull request merged**